server/errorstore/errors_test.go

```diff
@@ -248,13 +248,21 @@ func testErrorHandlerCollectsErrors(t *testing.T, pool fleet.RedisPool, wd strin
     \]
   \}`, wd, wd)), errors[0])
 
-	// and then errors are gone
-	errors, err = eh.Flush()
+	errors, err = eh.Retrieve(flush)
+	if flush {
+		require.NoError(t, err)
```

Comment:
`require.NoError(t, err)` in both branches.

Reply:
do you mean that since the check is used in both branches we could avoid the repetition? If yes, I agree but I want to make sure I get your comment before pushing the changes.

Reply:
Yes. Just a nitpick.

Reply:
pushed a fix and looks way better now. Thanks!